Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -776,7 +776,7 @@ def rasterize_thread(browser, chrome_port, path: str,
             raise DemistoException(f'Unsupported rasterization type: {rasterize_type}.')
 
 
-def perform_rasterize(path: str | list[str],
+def perform_rasterize(paths: str | list[str],
```

Comment:
Keep it single. For all you know, it can be only one.
```suggestion
def perform_rasterize(path: str | list[str],
```

Reply:
done